src/App.tsx

```diff
@@ -25,26 +29,40 @@ function Channels(): JSX.Element {
           newChannel.title = data.result.title;
           newChannel.link = `https://t.me/${channelName}`;
           newChannel.description = data.result.description ? data.result.description : "";
-          newChannel.pictureID = data.result.photo.big_file_id;
-          setChannelsArray((before) => [...before, newChannel]);
+          promisesPictures.push(fetch(`https://api.telegram.org/bot${API_KEY}/getFile?file_id=${data.result.photo.big_file_id}`)
+            .then(res => res.json())
+            .then(data => newChannel.pictureID = `https://api.telegram.org/file/bot${API_KEY}/${data.result.file_path}`)
+          )
         })
       );
+
+      promisesMembers.push(fetch(`https://api.telegram.org/bot${API_KEY}/getChatMembersCount?chat_id=@${channelName}`)
+        .then(res => res.json())
+        .then(data => newChannel.subscribers = data.result)
+      );
+
+      sortedArray.push(newChannel);
```

Comment:
what are you pushing here into the array it's without the subscribes, you should make this push in the `promise.then(() => {})`